docs/faq.md

```diff
@@ -171,7 +171,23 @@ mappings for your backend services. You can work around this problem using the
 `cluster_tag` property, as described in
 [emissary-ingress/emissary#3112](https://github.com/emissary-ingress/emissary/issues/3112).
 
+### HTTP 464 error with AWS
+
+If you're using the [Application Load Balancer support for gRPC][alp-aws-grpc], 
```

Comment:
```suggestion
If you're using the AWS [Application Load Balancer support for gRPC][alp-aws-grpc], 
```

Reply:
I'd argue the context should be clear from the headline, but don't mind making it extra clear.